socks-forwarder/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "socks-forwarder"
-version = "0.3.0"
+version = "0.5.0"
```

Comment:
Forgot to bump to 0.4.0 😮‍💨 ; not a huge issue as we are not even present on crates.io, so this Cargo version currently means nothing. Only the dpkg version really matters for now, but we should keep them in sync.